test/tensor_util.hpp

```diff
@@ -71,4 +71,114 @@ void operate_over_subtensor_impl(const data_operator_t<T>& r_data_operator,
     }
 }
 
+template <typename T>
+void output_tensor_to_csv(const tensor<T>& x, std::string filename)
+{
+    int dim = x.desc.GetSize();
```

Comment:
I have replaced `GetSize()` with `GetNumDims()` in https://github.com/ROCm/MIOpen/pull/2896. if CI passes, then this code is not used.